backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/index/impl/ElasticsearchIndexSearchTargetBuilder.java

```diff
@@ -50,14 +50,14 @@ public IndexSearchTarget build() {
 		Set<ElasticsearchIndexModel> indexModels = indexManagers.stream().map( ElasticsearchIndexManager::getModel )
 				.collect( Collectors.toCollection( LinkedHashSet::new ) );
 		ElasticsearchSearchTargetModel searchTargetModel = new ElasticsearchSearchTargetModel( indexModels );
-		return new ElasticsearchIndexSearchTarget( backend.getSearchContext(), searchTargetModel );
+		return new ElasticsearchIndexSearchTarget( searchBackendContext, searchTargetModel );
 	}
 
 	@Override
 	public String toString() {
 		return new StringBuilder( getClass().getSimpleName() )
 				.append( "[" )
-				.append( "backend=" ).append( backend )
+				.append( "searchBackendContext=" ).append( searchBackendContext )
```

Comment:
AFAICS it does not have a proper `toString()`, especially not one containing the name of the backend.

Reply:
Right. Done.